cloudbuild.yaml

```diff
@@ -14,10 +14,118 @@
 
 
 steps:
-# Uses the docker build step to build an image called cloud-run-pubsub-service
-- name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/cloud-run-pubsub-service', './philips_hue_integration_example']
+- id: 'build docker image'
+  # Uses the docker build step to build an image
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/${_IMAGE_NAME}', './philips_hue_integration_example']
```

Comment:
Is `-t` short for `--tag`? 

In general, CLIs provide one letter options to cut down on typing on the command line. However, it's better to use the longer version in code for readability. I know that you probably just followed the tutorial at https://cloud.google.com/cloud-build/docs/configuring-builds/create-basic-configuration, I don't really know why they use the abbreviations.

Reply:
Done